### DISCUSSION TITLE
2015 to Now: What Has Changed That Blows Your Mind?🫨 ?

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#142177] [Other Feature Feedback, Questions, & Ideas]
Can you believe Universe 2024 is the tenth Universe we’ve put on? A lot has changed since 2015, especially for developers 💽. 

Flashback to 2015, coding bootcamps were just beginning to emerge, Node.js and Angular grew in popularity, and cloud computing made software development more accessible around the globe. 

So, we want to know: What did you do in 2015 that seems unbelievable in 2024 🦄? And what do we have today that would’ve completely blown your mind 🤯 back in 2015?"

**Comment by maybeJosiah** [comment]
Well, I did not code in 2015 except NXTs so I can't say much for unbelievable now and I probably would not believe I would be working on a 3 or more dimensional system today as 2015 was maybe or maybe not, maybe.

**Comment by ScottV12** [comment]
You Mr maybe

**Comment by Mohataseem89** [comment]
In 2015 I was barely getting started with coding, and the idea of building full apps or working with AI tools felt impossible. Seeing how quickly development tools evolved since then still blows my mind.
